mungegithub/mungers/submit-queue.go

```diff
@@ -1074,18 +1082,72 @@ func (sq *SubmitQueue) cleanupOldE2E(obj *github.MungeObject, reason string) {
 
 }
 
-func priority(obj *github.MungeObject) int {
-	// jump to the front of the queue if you don't need retested
-	if obj.HasLabel(retestNotRequiredLabel) || obj.HasLabel(retestNotRequiredDocsOnlyLabel) {
-		return retestNotRequiredMergePriority
+const (
+	criticalFixPrio = iota
```

Comment:
This priority ordering is duplicated in a few places, making future changes more error-prone. Could you put the labels in a string slice and iterate through them for labelPriority and serveMergeInfo?

    labelPriority = []string{criticalFixLabel, retestNotRequiredLabel, ...}

    func labelPriority(obj *github.MungeObject) int {
        for i, label := range labelPriorities {
            if obj.hasLabel(label) {
                return i
            }
        }
        return len(labelPriorities)
    }